CMSgov/cms-carts-seds
# Wyoming 2022 ETL

### Description
We received a help desk ticket regarding Wyoming's 2022 report; they need to change the program type, but it is read-only.

This script will make that field writeable, when invoked through AWS Lambda.

### Related ticket(s)
CMDCT-3085

---
### How to test
1. Make sure your cloud environment has a report for Wyoming in 2022, and that the program type question in Basic State Information is readonly.
2. Run the script, by testing the lambda in AWS
3. Note that the question has become editable

-or-
1. Find a report in your local environment which has a readonly program type
2. Edit the script to point to that report (the `[state]-[year]` ID is on line 16)
3. Run the script, by `serverless invoke` (https://www.serverless.com/framework/docs/providers/aws/cli-reference/invoke-local)
4. Note that the question has become editable
5. Revert your change to the script

### Important updates
n/a

---
### Author checklist
